pkgs/by-name/ro/rosa/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles, stdenv, testers, rosa, go }:
+
+buildGoModule rec {
+  pname = "rosa";
+  version = "1.2.35";
+
+  src = fetchFromGitHub {
+    owner = "openshift";
+    repo = "rosa";
+    rev = "v${version}";
+    hash = "sha256-QMoUies5siHA9AlXLilz1/i6YTE8Z4HHO1BsyJnRkrc=";
+  };
+  vendorHash = null;
+
+  ldflags = [ "-s" "-w" ];
+
+  __darwinAllowLocalNetworking = true;
+
+  nativeBuildInputs = [ installShellFiles ];
+  postInstall = ''
+    installShellCompletion --cmd rosa \
+      --bash <($out/bin/rosa completion bash) \
+      --fish <($out/bin/rosa completion fish) \
+      --zsh <($out/bin/rosa completion zsh)
+  '';
+
+  preCheck = ''
+    ln -s $GOPATH/bin/rosa rosa
+  '';
+
+  passthru.tests.version = testers.testVersion {
+    package = rosa;
+    command = "rosa version";
+  };
+
+  meta = with lib; {
+    description = "CLI for the Red Hat OpenShift Service on AWS";
+    license = licenses.asl20;
+    homepage = "https://github.com/openshift/rosa";
+    maintainers = with maintainers; [ jfchevrette ];
+    platforms = go.meta.platforms;
```

Comment:
```suggestion
```
that is already being set

Reply:
I didn't know `buildGoModule` already set that. Thank you for pointing it out!